lib/ansible/modules/git.py

```diff
@@ -43,23 +43,28 @@
         default: "HEAD"
     accept_hostkey:
         description:
-            - If C(yes), ensure that "-o StrictHostKeyChecking=no" is
-              present as an ssh option.
+            - Will ensure or not that "-o StrictHostKeyChecking=no" is present as an ssh option.
+            - Be aware that this disables a protection against MITM attacks.
+            - Those using OpenSSH >= 7.5 might want to set I(ssh_opt) to 'StrictHostKeyChecking=accept-new'
+              instead, it does not remove the MITM issue but it does restrict it to the first attempt.
```

Comment:
The other sections are not using bullet-points, for consistency could the update be written in the same manner ?

Reply:
not bullet points, description is always a list, this is data type. Also making very separate points.